bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -0,0 +1,353 @@
+        sep = len(bi) - args.loudest_keep
+        sep = 0 if sep < 0 else sep
+
+        bsort = numpy.argpartition(c[bi], sep)
+        bl_int = bi[bsort[0:sep]]
+        bh = bi[bsort[sep:]]
+        del bsort
+        del bi
+    elif args.loudest_keep_value:
+        bh = bi[c[bi] > args.loudest_keep_value]
+        bl_int = bi[c[bi] <= args.loudest_keep_value]
+    else:
+        bh = bi
+
+    if args.decimation_factor:
+        bl = bl_int[slide[bl_int] % args.decimation_factor == 0]
+    else:
+        bl = []
+
+    ti = numpy.concatenate([bl, bh, fi]).astype(numpy.uint32)
+    logging.info('%s after decimation' % len(ti))
+
+    gs={}
+    for ifo in ids:
+        gs[ifo]=ids[ifo][ti]
+    del ids
+
+    # In the first for cycle the time and trigger_id keys will not exist,
+    # we need to create them, otherwise add new items into data.
+    checkstring = '%s/time' % ReadByTemplate(args.trigger_files[0]).ifo
+    if checkstring in data:
+        for ifo in gs:
+            addtime=times[ifo][gs[ifo]]
+            addtriggerid=tids[ifo][gs[ifo]]
+            data['%s/time' % ifo] += [addtime]
+            data['%s/trigger_id' % ifo] += [addtriggerid]
+    else:
+        for ifo in gs:
+            addtime=times[ifo][gs[ifo]]
+            addtriggerid=tids[ifo][gs[ifo]]
+            data['%s/time' % ifo] = [addtime]
+            data['%s/trigger_id' % ifo] = [addtriggerid]
+    data['stat'] += [c[ti]]
+    dec_fac = numpy.repeat([args.decimation_factor, 1, 1],
+                           [len(bl), len(bh), len(fi)]).astype(numpy.uint32)
+    data['decimation_factor'] += [dec_fac]
+    data['timeslide_id'] += [slide[ti]]
+    data['template_id'] += [numpy.zeros(len(ti), dtype=numpy.uint32) + tnum]
+
+if len(data['stat']) > 0:
+    for key in data:
+        data[key] = numpy.concatenate(data[key])
+
+if args.cluster_window and len(data['stat']) > 0:
+    timestring0 = '%s/time' % ReadByTemplate(args.trigger_files[0]).ifo
+    timestring1 = '%s/time' % ReadByTemplate(args.trigger_files[1]).ifo
+    cid = coinc.cluster_coincs(data['stat'], data[timestring0], data[timestring1],
+                               data['timeslide_id'], args.timeslide_interval,
+                               args.cluster_window)
+
+logging.info('saving coincident triggers')
+f = h5py.File(args.output_file, 'w')
+if len(data['stat']) > 0:
+    for key in data:
+        var = data[key][cid] if args.cluster_window else data[key]
+        f.create_dataset(key, data=var,
+                         compression='gzip',
+                         compression_opts=9,
+                         shuffle=True)
+
+f['segments/coinc/start'], f['segments/coinc/end'] = veto.segments_to_start_end(coinc_segs)
+
+for t in trigs:
+    f['segments/%s/start' % trigs[t].ifo], f['segments/%s/end' % trigs[t].ifo] = trigs[t].valid
+
+f.attrs['timeslide_interval'] = args.timeslide_interval
+f.attrs['num_of_ifos'] = len(args.trigger_files)
+
+for i in range(len(args.trigger_files)):
```

Comment:
Do we need ifo_1 etc stored? What does this map to? What we might want stored is the pivot and fixed ifo as attributes though.